cli/src/scanner.rs

```diff
@@ -296,227 +296,3 @@ impl EnvScanner {
         envs
     }
 }
-
-#[cfg(test)]
-mod tests {
```

Comment:
moved to the [`test` module](https://doc.rust-lang.org/book/ch11-03-test-organization.html#the-tests-module-and-cfgtest), mostly so `setup_repo_with_commit` can be reused with other tests